core/platform/search/elastic_search_services_test.py

```diff
@@ -17,6 +17,7 @@
 """Tests for the python elastic search wrapper."""
 
 from __future__ import annotations
+from core.domain import search_services
```

Comment:
Move this together with the imports on lines 22 and 23

Reply:
Done